Code/Core/FileIO/PathUtils.cpp

```diff
@@ -290,4 +290,39 @@
     outRelativeFileName += pathB;
 }
 
+// GetBaseName
 //------------------------------------------------------------------------------
+
+/* static */ void PathUtils::GetBaseName( const AString& pathName, AString& baseName )
+{
+    const char* slashPos = nullptr;
+    if ( nullptr != ( slashPos = pathName.FindLast( NATIVE_SLASH ) ) )
```

Comment:
The two lines above line could be simplified to:
```
if ( const char * slashPos = pathName.FindLast( NATIVE_SLASH ) )
```

Reply:
Fixed